homeassistant/components/emoncms/__init__.py

```diff
@@ -21,6 +24,39 @@ async def async_setup_entry(hass: HomeAssistant, entry: EmonCMSConfigEntry) -> b
         entry.data[CONF_API_KEY],
         session=async_get_clientsession(hass),
     )
+    emoncms_unique_id = await emoncms_client.async_get_uuid()
+    message = f"uuid for entry {entry.entry_id} is {entry.unique_id}"
+    LOGGER.debug(message)
```

Comment:
Single-use variable `message`

Note: there is a lot of debug messages, which might be useful for development purposes but I'm not sure they are useful to keep in the code permanently.

Reply:
debug message removed